conf/application.conf

```diff
@@ -53,7 +53,7 @@ controllers {
   }
 }
 
-play.filters.headers.contentSecurityPolicy = "default-src 'self' 'unsafe-inline' 'unsafe-eval' localhost:9032 localhost:9250 localhost:9310 www.google-analytics.com assets.publishing.service.gov.uk localhost:8291 *.analytics-egain.com stats.g.doubleclick.net www.google-analytics.com data:"
+play.filters.headers.contentSecurityPolicy = "default-src 'self' 'unsafe-inline' 'unsafe-eval' localhost:9032 localhost:9250 localhost:9310 www.google-analytics.com assets.publishing.service.gov.uk localhost:8291 *.analytics-egain.com stats.g.doubleclick.net www.google-analytics.com *.optimizely.com data:"
```

Comment:
We need to ensure that the CSP is updated in other environments to allow optimizely through, feel free to remove this if you want and create a new ticket to allow Optimizely through CSP

Reply:
will create a different ticket

Reply:
https://jira.tools.tax.service.gov.uk/browse/DDCNLS-7157

Have also linked this to the ticket DDCNLS-7127